docs/ko/docs/advanced/response-change-status-code.md

```diff
@@ -0,0 +1,33 @@
+# 응답 - 상태 코드 변경
+
+기본 [응답 상태 코드 설정](../tutorial/response-status-code.md){.internal-link target=_blank}이 가능하다는 걸 이미 알고 계실 겁니다.
+
+하지만 경우에 따라 기본 설정과 다른 상태 코드를 반환해야 할 때가 있습니다.
+
+## 사용 예
+
+예를 들어 기본적으로 HTTP 상태 코드 "OK" `200`을 반환하고 싶다고 가정해 봅시다.
+
+하지만 데이터가 존재하지 않으면 이를 새로 생성하고, HTTP 상태 코드 "CREATED" `201`을 반환하고자 할 때가 있을 수 있습니다.
+
+이때도 `response_model`을 사용하여 반환하는 데이터를 필터링하고 변환할 수 있습니다.
+
+이런 경우에는 `Response` 파라미터를 사용할 수 있습니다.
+
+## `Response` 파라미터 사용하기
+
+*경로 작업 함수*에 `Response` 타입의 파라미터를 선언할 수 있습니다 (쿠키와 헤더에 대해 선언하는 것과 유사하게).
```

Comment:
```suggestion
*경로 작동 함수*에 `Response` 타입의 파라미터를 선언할 수 있습니다 (쿠키와 헤더에 대해 선언하는 것과 유사하게).
```
Path operation should be ```경로 작동```
ref:https://github.com/fastapi/fastapi/discussions/3167#discussioncomment-721448